collection-scripts/logs/gather_logs_pvr

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Can we rename this file to `gather_logs_pv_restore` and the previous file to `gather_logs_pv_backup` ?

Reply:
Sorry just realized that this is not for user consumption. Ignore my previous comment